text/0000-stack-metadata.md

```diff
@@ -0,0 +1,95 @@
+# Meta
+[meta]: #meta
+- Name: Additional Stack Metadata
+- Start Date: 2020-05-12
+- Author(s): kvedurmu
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+We currently expose the Stack ID and Mixins via Labels on the build/run images. It would be great if we could leverage Labels to allow stack authors to specify additional image metadata (e.g OS Distro, Maintainer, Stack Version, etc.). The purpose of this RFC is to enumerate on additional metadata we want to expose in our stack images and standardize on these labels.
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this?  
+Exposing additional metadata in stack images will make it easier for end users to debug stack related issues. Also, by exposing this metadata, CNB platforms such as kpack can leverage this information to improve their UX. 
+- What is the expected outcome?  
+We converge on metadata we want to expose in stack images via labels, and all stack authors eventually support these.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This provides a high level overview of the feature.  
+
+- Define the target persona: stack author  
+- Explaining the feature largely in terms of examples.  
+The following labels will be added to stack images:  
```

Comment:
Somewhere in the document, it should be called out that these are either optional or required.  I’m of the opinion that they should _all_ be optional, but don’t feel strongly about that.

Reply:
optional but strongly encouraged?

Reply:
My policy is MAY means WON'T and SHOULD means PROBABLY WON'T.  But  SHOULD is probably the right strength :)